Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.cs

```diff
@@ -438,13 +437,12 @@ private FileSystemWatcher CreateFileSystemWatcher(string dir)
             {
                 InternalBufferSize = 1024 * 4,  // 4k is minimum buffer size
                 IncludeSubdirectories = true
+                /* don't need to set NotifyFilters, since default LastWrite, FileName, and DirectoryName works for us */
```

Comment:
>/* don't need to set NotifyFilters, since default LastWrite, FileName, and DirectoryName works for us */ [](start = 16, length = 104)

I found this comment confusing.  Is not setting `NotifyFilter` so unconventional that it requires as explanation?  If so, I'd just say "the default value" rather than listing the exact filters.

Reply:
It took me a while to figure out what the actual default was... so being explicit seemed helpful. I can reword

Reply:
In this case, it's documented on MSDN.

---
In reply to: [159307551](https://github.com/Microsoft/nodejstools/pull/1827#discussion_r159307551) [](ancestors = 159307551)